docs/license.md

```
@@ -65,19 +62,19 @@ Copyright (c) 2024 ニケ
 
 ## ライセンスの選択
 
-1. **個人利用、教育目的、非営利目的**での使用の場合は、MITライセンスが自動的に適用されます。
-2. **商用目的**での使用の場合は、適切な商用ライセンスを購入する必要があります。
+1. **個人利用、教育目的、非営利目的、展示・デモンストレーション**での利用の場合は、無償でご利用いただけます。
+2. **商用目的**での利用の場合は、適切な商用ライセンスを購入する必要があります。
 
-ライセンスの選択に迷う場合や、ご質問がある場合は、support@aituberkit.com までご連絡ください。
+ライセンスの選択に迷われた場合やご質問がある場合は、**support@aituberkit.com** までお気軽にお問い合わせください。
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)